src/components/SkeletonBodyText/README.stories.mdx

```diff
@@ -0,0 +1,44 @@
+import dedent from 'ts-dedent'
+import {Meta, Story, Canvas, ArgsTable} from '@storybook/addon-docs'
+import {SkeletonBodyText} from '@/polaris-vue';
+
+<Meta
+    title="Components / Feedback indicators"
+    component={SkeletonBodyText}
+/>
+
+export const Template = (args) => ({
+    components: { SkeletonBodyText },
+    setup() {
+        return { args };
+    },
+    template: `
+      <SkeletonBodyText v-bind:lines="args.lines"/>
+    `,
+});
+
+# Skeleton body text
+
+Skeleton body text is used to provide a low fidelity representation of content before it appears on the page, and improves load times perceived by merchants. Can be used for content in or outside of a card.
+
+<Canvas>
+    <Story
+        name="SkeletonBodyText"
+        args={{
+            lines: 3,
+        }}
+        parameters={{
+            docs: {
+                source: {
+                    code: dedent`
+                        <SkeletonBodyText :lines="3"/>
```

Comment:
Don't fix this. Because it's flexible for users. Try `<SkeletonBodyText v-bind="props" />`